app/battle/batlle.py

```diff
@@ -0,0 +1,18 @@
+from app.knight.knight import Knight

```

Comment:
As for me, It is not good to name the file the same as the folder.
I recommend you to rename it to model or entity